File: Python Code/Daily_Flash_Code_0/Week9/17Mar/49-DailyFlash_Solutions/17-Mar/Python/prog3.py
def isAng(num):
    s = 0
    temp = num
    while(temp != 0):
        rem = temp % 10
        s += rem * rem * rem
        temp = int(temp / 10)
    if(s == num):
        return True
    return False
while(True):
    num = int(input("Enter num : "))
    if(isAng(num)):
        break
    print(num)

File: Python Code/Daily_Flash_Code_0/Week9/17Mar/49-DailyFlash_Solutions/17-Mar/Java/prog3.java
import java.util.Scanner;

class Demo {
	static int isAng(int num) {
		int sum = 0, temp = num, rem;
		
		while(temp != 0) {
			rem = temp % 10;
			sum += rem * rem * rem;
			temp /= 10;
		}
		if(sum== num)
			return 1;
		return 0;
	}	
	public static void main(String[] args) {
		int num = 0, f = 0;
		Scanner sc = new Scanner(System.in);
		do {
			if(f != 0)
				System.out.println(num);	
			num = sc.nextInt();
			f = 1;
		} while(isAng(num) != 1);
	}
}
